.eslintignore

```diff
@@ -0,0 +1,2 @@
+dist/**/*
```

Comment:
Should we keep the `dist` folder under version control? I.e. auto-generated code tends to add noise to the review process and weight to the commit history.

Whereas, it is possible to configure static hosting services, like Netlify, to automatically build and host the document from source files.

Reply:
Yes, the dist folder and all autogenerated files (schemas and full OpenAPI specification compiled from it) are not put into version control.

`npm run dist`builds schemas, full Open API specification and any relevant static files to be deployed.

`npm run deploy`deploys to GitHub pages using `gh-pages` npm package.

I havent investigated yet but this way it is possible to use GitHub Actions or Netlify to deploy it without having a need to have auto-generated files being under version control.

Reply:
OK, thanks for clarifying. Apologies for the confusion.